tools/monorepo-utils/src/changefile/index.ts

```diff
@@ -91,8 +91,11 @@ const program = new Command( 'changefile' )
 				`Temporary clone of '${ headOwner }/${ name }' created at ${ tmpRepoPath }`
 			);
 
-			Logger.notice( `Checking out remote branch ${ branch }` );
-			await checkoutRemoteBranch( tmpRepoPath, branch );
+			// If a pull request is coming from a contributor's fork's trunk branch, we need to checkout the remote branch.
```

Comment:
The code below seems to be doing the opposite of the comment. In other words, if a pull request **isn't** coming from a contributor's fork's trunk branch...

I'm not sure I follow what would happen in the case that the remote branch were `trunk`. What about cases where we have a branch on origin named the same as the contributors remote branch, are those problematic or is it only trunk?

Reply:
>The code below seems to be doing the opposite of the comment. In other words, if a pull request isn't coming from a contributor's fork's trunk branch...

Oops, the comment is backwards. Fixed in https://github.com/woocommerce/woocommerce/pull/39306/commits/6c2fda63b62ef2873a9d387f7d6945af20c918fc

Reply:
>I'm not sure I follow what would happen in the case that the remote branch were trunk
It will be fine, we just want to avoid re-creating a new branch also named `trunk` -  a git error will occur

>What about cases where we have a branch on origin named the same as the contributors remote branch, are those problematic or is it only trunk?

This will also be fine, the changefile commit will be pushing to the contributor's branch on their fork